src/com/hms_networks/americas/sc/thingworx/TWConnectorConsts.java

```diff
@@ -68,20 +68,16 @@
   public static final int CONNECTOR_CONFIG_JSON_INDENT_FACTOR = 3;
 
   // Config Keys
-  /** The configuration file JSON key for the Thingworx IP address. */
-  public static final String CONNECTOR_CONFIG_TW_IP_KEY = "ThingworxIP";
+  /** The configuration file JSON key for the Thingworx Url address. */
+  public static final String CONNECTOR_CONFIG_TW_URL_KEY = "ThingworxUrl";
```

Comment:
Per @TomKimsey's comment, this should be edited from "ThingworsUrl" to something like "ThingworxAddress" or "ThingworxAddr"

Reply:
As I mentioned in my comment, I dont see "Address" as a great choice as its meaning is somewhat vague.

Reply:
To clarify on my initial comment about keeping ThingworxIP that would be in addition to ThingworxUrl.

A user could have the config

```
{
   "LogLevel": 1,
   "ThingworxUrl": "https://setme.biz",
   "AppKey": "Set-This-Value"
}
```
or
```
{
   "LogLevel": 1,
   "ThingworxIp": "192.168.0.1",
   "AppKey": "Set-This-Value"
}
```

if they had both just use the url (if not empty) and print an error.

Reply:
IP address is also misleading because "192.168.0.1:8080" will also work and it's not strictly an IP address.


Also, RFC that defines URL state the following about the host portion of URL: 
_the fully qualified domain name of a network host, or its IP address_ 

Is "ThingworxBaseUrl" a better choice?

Reply:
I think what you have is fine but maybe add explanation in the readme that the host portion of the url can be an ip address as well.

Reply:
Done. I also talk about choice of scheme and the impact on TLS usage.